### PR TITLE
support modules

### PR DESCRIPTION
**Pull request by miparnisari**
*Issue #, if available:* N/A

*Description of changes:* add support for newly-launched modules

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by kddejong — APPROVED** [review]

File: src/cfnlint/rules/resources/Configuration.py [hunk 1]
```diff
@@ -62,7 +62,6 @@ def _check_resource(self, cfn, resource_name, resource_values):
             ))
 
         resource_type = resource_values.get('Type', '')
-        check_attributes = []
```
Comment (PatMyron) [inline src/cfnlint/rules/resources/Configuration.py:65]: @miparnisari why was this removed?

Comment (miparnisari) [reply src/cfnlint/rules/resources/Configuration.py:65]: PyCharm reported that it was unused
